pkg/providers/config.go

```diff
@@ -79,8 +79,9 @@ func (m *ConfigMapConfigManager) buildDefaultConfigMap() *v1.ConfigMap {
 			Namespace: m.providerConfigMapNamespace,
 		},
 		Data: map[string]string{
-			"managed":  "{\"blobstorage\":\"aws\", \"redis\":\"aws\", \"postgres\":\"aws\"}",
-			"workshop": "{\"blobstorage\":\"openshift\", \"redis\":\"openshift\", \"postgres\":\"openshift\"}",
+			"openshift": "{\"blobstorage\":\"openshift\", \"redis\":\"openshift\", \"postgres\":\"openshift\"}",
+			"aws":       "{\"blobstorage\":\"aws\", \"redis\":\"aws\", \"postgres\":\"aws\"}",
+			"gcp":       "{\"blobstorage\":\"gcp\", \"redis\":\"gcp\", \"postgres\":\"gcp\"}",
```

Comment:
I agree that moving to this model makes more sense but my only concern would be if the config map already exists. If the provider config map is present with `managed` configuration what will happen when updating CRO?

Reply:
Ah looks like value in the CR takes precedence so will continue to use `managed` in CR until/unless we override it.